### PR TITLE
DO-1844 Store unfinished construction

### PR DESCRIPTION
**Pull request by OlexandrTopuzov**

### CODE REVIEW

**Review by OlexandrTopuzov — COMMENTED** [review]
@roman-tiukh 
@EmmaSamoilova

File: business_register/models/declaration_models.py [hunk 1]
```diff
@@ -767,6 +767,13 @@ class Property(DataOceanModel):
         null=True,
         help_text=_('valuation of the property')
     )
+    is_unfinished = models.BooleanField(
+        _('is unfinished constractions'),
+        null=True,
+        blank=True,
+        default=None,
+        help_text=_('is unfinished constractions'),
+    )
```
Comment (roman-tiukh) [inline business_register/models/declaration_models.py:776]: why we need this fied?

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/models/declaration_models.py (hunk 1 above)
Comment (OlexandrTopuzov) [reply business_register/models/declaration_models.py:776]: so said to do Єhor, I suppose that the structures in the subsequent can beat the completion

Comment (roman-tiukh) [reply business_register/models/declaration_models.py:776]: BooleandField should have default value and dont have null=True

Comment (OlexandrTopuzov) [reply business_register/models/declaration_models.py:776]: the fact is that we need to apply this field only to type 6 and if the default is true, then all types will be true

**Comment by roman-tiukh** [comment]
wait for Iegor`s review

**Review by roman-tiukh — APPROVED** [review]

File: business_register/converter/declaration.py [hunk 2]
```diff
@@ -615,7 +615,28 @@ def save_luxury_item(self, luxuries_data, declaration):
 
     # TODO: implement as save_property()
     def save_unfinished_construction(self, unfinished_construction_data, declaration):
-        pass
+        for data in unfinished_construction_data:
+            additional_info = data.get('objectType')
+            # TODO: add country
+            country = self.find_country(data['country'])
```
Comment (EmmaSamoilova) [inline business_register/converter/declaration.py:621]: There may be a KeyError if the 'country' key is missing. Or is 'country' in all declarations?

Comment (EmmaSamoilova) [inline business_register/converter/declaration.py:621]: There may be a KeyError if the 'country' key is missing. Or is 'country' in all declarations?

**Pull request merged**